petstore/clients/dotnet/sample/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using PetStore;
+using PetStore.Models;
+using System.ClientModel;
+
+var petClient = new PetStoreClient(new Uri("http://localhost:5118")).GetPetsClient();
+
+// create a pet
+var pet = await petClient.CreateAsync(new PetCreate("Kiwi", 5, 0));
```

Comment:
Do we support implicit conversion from `ClientResult<T>` to `T`?

Reply:
if not, maybe we should use `petCreateResult`?

Reply:
we have an implicit operator but it will not be invoked if we use var.
Let me change this to petCreateResult.

Reply:
I just remembered that we have a rule that we should avoid of using `var` in our samples.
Therefore I have updated the code to avoid using var.